gzdoom/gzdoom.json

```diff
@@ -4,6 +4,9 @@
     "config-opts": [
         "-DCMAKE_BUILD_TYPE=Release"
     ],
+    "build-options" : {
+        "cxxflags": "-Wp,-U_GLIBCXX_ASSERTIONS"
```

Comment:
if the code crashes with these assertions the code should be fixed. These assertions exists for a reason 100% of their triggers are bug that are usually obscured.

Reply:
I'm discussing the issue Upstream. If they choose to distribute GZDoom with these known issues, then I don't think it's my job to decide otherwise. I'll try to keep this Flathub as close as possible to the official GZDoom binary.